htsget.md

```diff
@@ -57,13 +61,13 @@ _object_
 Container for response object.
 <table>
 <tr markdown="block"><td>
-`error`  
+`error`
```

Comment:
Don't remove the trailing whitespace on these lines. They are [kramdown line breaks](https://kramdown.gettalong.org/quickref.html#paragraphs). (I think the two-backslashes notation didn't exist at the time and might be worth converting to, but that would be a separate change.)

Reply:
Sorry you're right. I shouldn't have removed them. I think it's my text editor setting that strips trailing whitespace out. On reflection I'll revert the htsget changes and let those guys introduce in TOC if they want as you've suggested.